src/cpp/client/ProxyClient.cpp

```diff
@@ -36,12 +36,14 @@ namespace uxr {
 
 ProxyClient::ProxyClient(
         const dds::xrce::CLIENT_Representation& representation,
-        Middleware::Kind middleware_kind)
+        Middleware::Kind middleware_kind,
+        std::unordered_map<std::string, std::string>&& properties)
     : representation_(representation)
     , objects_()
     , session_(SessionInfo{representation.client_key(), representation.session_id(), representation.mtu()})
     , state_{State::alive}
     , timestamp_{std::chrono::steady_clock::now()}
+    , properties_{properties}
```

Comment:
```suggestion
    , properties_(std::move(properties))
```
The move to the final destination (the class attribute) is missing